src/storm/generator/JaniNextStateGenerator.cpp

```diff
@@ -156,6 +158,7 @@ bool JaniNextStateGenerator<ValueType, StateType>::canHandle(storm::jani::Model
     features.remove(storm::jani::ModelFeature::DerivedOperators);
     features.remove(storm::jani::ModelFeature::Functions);  // can be substituted
     features.remove(storm::jani::ModelFeature::StateExitRewards);
+    features.remove(storm::jani::ModelFeature::TrigonometricFunctions);
```

Comment:
Is this true? If so, I think this should be covered in a test. 
If you are not sure, I would not allow this right now?

Reply:
Hello Sebastian,

I am using it in the smc extension we are developing in the CONVINCE, so I am quite convinced this is working.
However, having one more test never hurts! :)

Some references about the smc use case:
- test: https://github.com/convince-project/smc_storm/blob/main/test/test_models.cpp#L58
- where this is used: https://github.com/convince-project/smc_storm/blob/main/src/model_checker/state_generation.cpp#L42

Reply:
If you want to make sure that we dont break this, a test definitively does not hurt :-)

Reply:
I adapted the test_trigonometry jani file to not expand forever and added a test for it in the `ExplicitJaniModelBuilderTest` gtest, (that is the only place I found with similar tests).
The new unit test can be found in this commit: https://github.com/moves-rwth/storm/pull/556/commits/6b54f7c5a1b5e01fb5ab0c645bfde906d4a87d03
On doing it, I found that transcendental numbers were not handled correctly in the constant variables bounds, but now they are handled, too.